config/secrets.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
Add above
```suggestion
if (process.env.NODE_ENV !== 'production') {
  require('dotenv').config()
}
module.exports = {
```

So from other envs it should get from the `.env` file and in production it will read from production enviromnent secret from production. So in this way we will have the proper env secrets for each environment